hasher-matcher-actioner/src/OpenMediaMatch/tests/test_e2e_workflow.py

```diff
@@ -39,7 +39,7 @@ def test_add_hashes_and_find_match(app: Flask, client: FlaskClient):
     # Now match
     resp = client.get(f"/m/lookup?signal_type=pdq&signal={hashes[-1]}")
     assert resp.status_code == 200
-    assert resp.json == {"TEST_BANK": [{"bank_content_id": 16, "distance": 0}]}
+    assert resp.json == {"TEST_BANK": [{"bank_content_id": 16, "distance": "0"}]}
```

Comment:
This is the net result of this change - I can try and do something to detect when there's a primitive type in the distance slot and return the correctly typed JSON if that would be preferable.